packages/react-slider/src/components/Slider/stories/SliderDefault.stories.tsx

```diff
@@ -0,0 +1,12 @@
+import * as React from 'react';
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-ignore
+import { Label } from '@fluentui/react-label';
+import { Slider } from '../../../index'; // codesandbox-dependency: @fluentui/react-slider ^9.0.0-beta
```

Comment:
```suggestion
import { Slider } from '../../../index';
```

aaaaand we no longer need these :)